src/features/game/lib/level.ts

```diff
@@ -151,19 +151,30 @@ export const findLevelRequiredForNextSkillPoint = (
 ): BumpkinLevel | undefined => {
   const level = getBumpkinLevel(experience);
 
-  if (Number(level) >= MAX_BUMPKIN_LEVEL) return undefined;
+  if (level >= MAX_BUMPKIN_LEVEL) {
+    return;
+  }
 
-  const currentSkillPoints = SKILL_POINTS[level];
-  const skillPointToLevels = flipObject(SKILL_POINTS);
+  let nextLevelWithSkillPoint = (level + 1) as BumpkinLevel;
+  // Skip levels that give no skill points
+  while (
```

Comment:
This is all working well but this part of the code is not very easy to read. Can you think of a way to refactor this to be a little easier to understand?

Reply:
How about this, @spencerdezartsmith?

```
```

Reply:
Replaced 'while' with 'for in' loop.

Reply:
@AzaroQL My versions quite a bit simpler.  Dunno if you saw it.

Reply:
> 

Both methods looks good to me, yours is less lines tho.